src/pretix/control/views/organizer.py

```diff
@@ -2311,6 +2311,14 @@ def get_context_data(self, **kwargs):
             o.icnt = annotated.get(o.pk)['icnt']
             o.sales_channel_obj = scs[o.sales_channel]
 
+        ctx["lifetime_spending"] = (
+            self.get_queryset()
+            .filter(status__in=[Order.STATUS_PAID, Order.STATUS_PENDING])
```

Comment:
The tooltip above says paid, but this still includes unpaid. I think we settled on just the paid ones :)

Reply:
Did this somehow get lost in all my commits? Well, good catch!